templates/spin_sidebar.hbs

```diff
@@ -89,7 +89,7 @@
                     <li><a {{#if (active_content request.spin-path-info "/spin/kv-store-api-guide" )}} class="active"
                             {{/if}} href="{{site.info.base_url}}/spin/kv-store-api-guide">Key Value Store</a></li>
                     <li><a {{#if (active_content request.spin-path-info "/spin/serverless-ai-api-guide" )}} class="active"
-                            {{/if}} href="{{site.info.base_url}}/spin/serverless-ai-api-guide">Serverless AI</a></li>
+                            {{/if}} href="{{site.info.base_url}}/spin/serverless-ai-api-guide">Build a Sentiment Analysis API using Serverless AI Inferencing</a></li>
```

Comment:
This appears to be the LLM API reference.

Reply:
You are right. My bad. I'll make the change now